04-Enumerable/Enumerable/Task.cs

```diff
@@ -35,9 +37,10 @@ public class Task {
         ///   {"a","aa","aaa" } => { 1, 2, 3 }
         ///   {"aa","bb","cc", "", "  ", null } => { 2, 2, 2, 0, 2, 0 }
         /// </example>
-        public IEnumerable<int> GetStringsLength(IEnumerable<string> data) {
+        public IEnumerable<int> GetStringsLength(IEnumerable<string> data)
+        {
             // TODO : Implement GetStringsLength
-            throw new NotImplementedException();
+            return data.Select(x => x == null ? 0:x.Length);
```

Comment:
Null propagating?
https://www.codingame.com/playgrounds/5107/null-propagating-operator-in-c